dataprofiler/profilers/float_column_profile.py

```diff
@@ -297,7 +297,7 @@ def _is_each_row_float(cls, df_series: pd.Series) -> List[bool]:
         :param df_series: series of values to evaluate
         :type df_series: pandas.core.series.Series
         :return: is_float_col
-        :rtype: list
+        :rtype: pandas.Series[bool]
```

Comment:
need to update this too based on type hint on output @Sanketh7

Reply:
Done.

Reply:
thanks

Reply:
not seeing this change ... @Sanketh7

Reply:
oops i forgot to push it